src/java.base/share/classes/jdk/internal/vm/annotation/JvmtiMountTransition.java

```diff
@@ -28,11 +28,14 @@
 import java.lang.annotation.*;
 
 /**
- * A method is annotated as "jvmti mount transition" if it starts
- * or ends virtual thread mount state transition (VTMS transition).
+ * A method may be annotated with JvmtiMountTransition to hint
+ * it is desirable to omit it from JVMTI stack traces.
+ * Normally, a method is annotated with @JvmtiMountTransition if it starts
+ * or ends Virtual Thread Mount State (VTMS) transition, so the thread
+ * identity is undefined or different at method entry and exit.
  *
  * @implNote
- * This annotation is only used for VirtualThread methods.
+ * This annotation is only used for the VirtualThread notifyJvmti*  methods.
```

Comment:
What about VirtualThread.switchToCarrierThread and VirtualThread.switchToVirtualThread ? They also have the annotation.

Reply:
We working to remove these two methods. I think the main thing for JvmtiMountTransition is that the interface description provides enough information to know when the annotation is needed.

Reply:
Good catch, thanks. These two methods are impacted by temporary VTMS transitions.
I've restored as it was before.